public/js/components/Editor.css

```diff
@@ -42,10 +41,6 @@
   background-color: var(--theme-body-background);
 }
 
-.CodeMirror-linenumber {
-  font-size: 11px;
-}
```

Comment:
Hmm setting a specific line-height (eg: 18px) doesn't give consistent result in Chrome and Firefox (Chrome is too short by 1px). 
#### Chrome : 
![debuggerchromelineheight](https://cloud.githubusercontent.com/assets/2848472/19130437/1fca72b2-8b19-11e6-903b-f0256fdf4924.jpg)
#### Firefox : 
![debuggerfirefoxlineheight](https://cloud.githubusercontent.com/assets/2848472/19130442/2341378c-8b19-11e6-83bd-ae3097e8b587.jpg)

Going back to the original css code, I get the gutter highlight misalignment in both Firefox and Chrome.  @jasonLaster do you only see this issue in Chrome?  

Setting the font-size to inherit makes the gutter font size consistent with the main panel for me and fixes the highlight height issue, not sure if that'll be the same case for you guys.